docs/README.md

```diff
@@ -0,0 +1,416 @@
+[merkle-patricia-tree](README.md)
+
+# merkle-patricia-tree
+
+## Index
```

Comment:
So I see quite a few items here that I don't think should be public and documenting them might give the impression they are public. We're exporting them from the individual files so that other files can use, but ideally only the things re-exported in `index.ts` should be considered public. I don't remember exactly if typedoc supported something like this, I remember I had struggled with this before..

Reply:
Gotcha, thanks. In `ethereumjs-util` I discovered `typedoc@next` has a library mode (https://github.com/TypeStrong/typedoc/issues/639) that is quite neat. I gave it a try here with some additional `exclude` params and seems like it's working well. I just pushed a commit if you would like to take a look.